WordPressKit.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '6.0.0-beta.1'
+  s.version       = '6.0.0-beta.2'
```

Comment:
As mentioned [here](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/567/files#r1081928178), maybe we need a major version bump?

Reply:
We might be able to get away with it since:

- Both [Jetpack/WordPress iOS](https://github.com/wordpress-mobile/WordPress-iOS/blob/27b5988f4b80b364626f3f8afdecb19573f38cf2/Podfile.lock#L613) and [WooCommerce iOS](https://github.com/woocommerce/woocommerce-ios/blob/2f7a8a74ee02ba9b74bb706e3a6679e997df642b/Podfile.lock#L88) still depend on 5.x
- 6.0.0 technically is still in beta, so new breaking changes, while not recommended, are still admissible

Reply:
@mokagio That makes sense, 6.0.0 wasn't released yet.

Reply:
I forgot Tony was AFK yesterday and today, too.

To move this along, I'm going to dismiss his request for changes on the basis of what discussed above and merge. That will also be useful to move forward with the WordPress Authenticator upgrade, see  https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/729 and https://github.com/wordpress-mobile/WordPress-iOS/pull/19908.

I'm aware of the risk of having missed something but I think given the small users-base WordPress Kit has–namely Jetpack/WordPress and WooCommerce via WordPress Authenticator–the cost of having an incorrect version in the wild is minimal.